physics/ephemeris.hpp

```diff
@@ -38,6 +39,8 @@ class Ephemeris {
   static std::nullptr_t constexpr kNoIntrinsicAcceleration = nullptr;
   using IntrinsicAccelerations = std::vector<IntrinsicAcceleration>;
   static IntrinsicAccelerations const kNoIntrinsicAccelerations;
+  static std::int64_t constexpr no_ephemeris_step_limit =
```

Comment:
This is called `step_limit` here and `max_steps` at line 155.  Let's try to be consistent.  I think we should stick to `max_steps` because we use it elsewhere.  How about `unlimited_max_ephemeris_steps`?